Added option to auto-join a default team

kurisu-gh opened the pull request:
Deployer can optionally set a default team for people to auto-join when they are added to the organization. 

Also cleaned up a few typos.

thundergolfer commented:
Thanks for your contribution. This looks like a good change. 

I've just merged another change that creates a small conflict in `index.slim`. Are you able to resolve that such that your change works with new `master`? 🙏

kurisu-gh commented:
Resolved the conflict to use your `index.slim` but I added my own small change to make the input box param to be `github-user` instead of `github` as I've changed the ruby app to access the username param that way. (Just to make it clear that it's the github username, and not say the github org name or team name).
